pkg/types/installconfig.go

```diff
@@ -141,6 +141,10 @@ type InstallConfig struct {
 	// GCP: "Mint", "Passthrough", "Manual"
 	// +optional
 	CredentialsMode CredentialsMode `json:"credentialsMode,omitempty"`
+
+	// BootstrapInPlace is the configuration for installing single node
+	// with bootstrap in place installation.
+	BootstrapInPlace *BootstrapInPlace `json:"bootstrapInPlace"`
```

Comment:
This is not a required field, and should be omitempty. This should also fix the unit tests.
```diff
-	BootstrapInPlace *BootstrapInPlace `json:"bootstrapInPlace"`
+	BootstrapInPlace *BootstrapInPlace `json:"bootstrapInPlace,omitempty"`
```